packages/client/src/pages/game/game.tsx

```diff
@@ -0,0 +1,133 @@
+            // eslint-disable-next-line
+            function fillTextWithSpacing(context: CanvasRenderingContext2D, text: string, x: number, y: number, spacing: number) {
+                const totalWidth = context.measureText(text).width + spacing * (text.length - 1);
+
+                const align = context.textAlign;
+                context.textAlign = 'left';
+                /* eslint-disable */
+                switch (align) {
+                    case 'right':
+                        x -= totalWidth;
+                        break;
+                    case 'center':
+                        x -= totalWidth / 2;
+                        break;
+                }
+                /* eslint-enable */
+
+                let offset;
+                let pairWidth;
+                let charWidth;
+                let charNextWidth;
+                let pairSpacing;
+                let char;
+                let charNext;
+
+                // eslint-disable-next-line
+                for (offset = 0; offset < text.length; offset = offset + 1) {
+                    char = text.charAt(offset);
+                    pairSpacing = 0;
+                    if (offset + 1 < text.length) {
+                        charNext = text.charAt(offset + 1);
+
+                        pairWidth = context.measureText(char + charNext).width;
+                        charWidth = context.measureText(char).width;
+                        charNextWidth = context.measureText(charNext).width;
+                        pairSpacing = pairWidth - charWidth - charNextWidth;
+                    }
+
+                    context.fillText(char, x, y);
+                    if (charWidth) {
+                        // eslint-disable-next-line
+                        x = x + pairSpacing + spacing + charWidth;
+                    }
+                }
+
+                context.textAlign = align;
+            }
+            ctx!.font = '19px inter';
+            ctx!.fillStyle = 'white';
+            const textH = 'A B C D E F G H I G';
+            // eslint-disable-next-line
+            [...Array(2).keys()].forEach(i => fillTextWithSpacing(ctx!, textH, 257 + 400 * i, 63, 6.6));
+
+            // render horizontal text
+            // eslint-disable-next-line
+            [...Array(2).keys()].forEach(i => {
+                for (let index = 0; index < 10; index += 1) {
+                    let x = 233 + 400 * i;
+                    if (index === 9) {
+                        x -= 7;
+                    }
+                    ctx!.fillText(String(index + 1), x, 93 + 30 * index);
+                }
+            });
+
+            // render ships
+            // eslint-disable-next-line
+            const shipsAmount = [4, 3, 2, 1];
+            ctx!.font = '32px inter';
+            // eslint-disable-next-line
+            [...Array(4).keys()].forEach(i => {
+                const image = new Image();
+                image.src = `../../../../images/ship_${i}.svg`;
```

Comment:
При сборке файлы изображений теряются. 
![image](https://github.com/rpserjo/praktikum-game/assets/20280355/df343000-cd93-49a5-9881-11d7e3f2fcce)

Лучше в папке /public создать директорию /sprites и положить изображения туда. И подключать их следующим образом:
```suggestion
                image.src = `./sprites/ship_${i}.svg`;
```